source/community/groups/framework.md

```diff
@@ -106,7 +104,7 @@ Technical Specification Groups are the main working vehicle for adding new APIs
 
 ### Technical Specification Group Requirements
 
-All members of a Technical Specification Group must agree to have any contributed work licensed under a CC-BY license, or similar.  Participants meeting this requirement may join at any time, without any prior approval process: the gateway is activity, not reputation.
+All members of a Technical Specification Group must agree to have any contributed work licensed under a CC-BY license.  Participants meeting this requirement may join at any time, without any prior approval process: the gateway is activity, not reputation.
```

Comment:
Any reason for the or similar to be removed?

Reply:
I removed it as I would really (like _really_) hate to have a spec derailed by lack of clarity around IP issues. If there's more than one license, then we need to track which contribution is being added under which IP framework and that seems like a total nightmare.  The specs are CC-BY, so that's what we should have people (implicitly) agreeing to.

Reply:
I don't disagree but should this be a separate pull request?

Reply:
Although it is documented here: https://iiif.io/api/annex/notes/disclaimer/

Reply:
Yeah, I think this falls under the idea of clarification to agree with the disclaimer

Reply:
Yup, I don't think it needs a separate PR, it's just resolving an inconsistency.